src/ABSmartly.Sdk/Context.cs

```diff
@@ -998,6 +1117,13 @@ public class ExperimentVariables
         public Experiment Data { get; set; }
         public List<Dictionary<string, object>> Variables { get; set; }
     }
+    
+    public class ContextCustomFieldValue
+    {
+        public String Name { get; set; }
```

Comment:
What's mapping the lower case names in the json to upper case in the code? Is it done automatically?